MAUI/Popup/styles.md

```diff
@@ -279,6 +279,34 @@ public partial class MainPage : ContentPage
 
 ![Displaying a .NET MAUI Popup with stroke customization](Images/styles/maui-popup-with-stroke-customization.png)
 
+## popup background
+
+The `SfPopup` allows to customize the background color of popup view using the `PopupBackground` property.
```

Comment:
The `SfPopup` allows to customize the background color of the popup view using the `PopupBackground` property.